cmd/localapiclient/localapiclient_test.go

```diff
@@ -0,0 +1,73 @@
+package localapiclient
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+)
+
+var ctx = context.Background()
+
+type BadStatusHandler struct{}
+
+func (b *BadStatusHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusBadRequest)
+}
+
+func TestBadStatus(t *testing.T) {
+	ctx, cancel := context.WithDeadline(ctx, time.Now().Add(2*time.Second))
+	client := New(&BadStatusHandler{})
+	defer cancel()
+
+	_, err := client.Call(ctx, "POST", "test", nil)
+
+	if err.Error() != "request failed with status code 400" {
+		t.Error("Expected bad status error, but got", err)
+	}
+}
+
+type TimeoutHandler struct{}
+
+var successfulReport = "successful report!"
+
+func (b *TimeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	time.Sleep(6 * time.Second)
+	w.Write([]byte(successfulReport))
+}
+
+func TestBugTimeout(t *testing.T) {
```

Comment:
Is "Bug" left over from when the `localapiclient` package still had bug reporting functionality in it?

Reply:
Yup, thanks for pointing this out!